backend/src/services/auth/auth.service.ts

```diff
@@ -29,4 +30,26 @@ const signupLocal = async (
   };
 };
 
-export { signupLocal };
+const signinLocal = async (user: User): Promise<SignInResponseData> => {
+  const { email } = user;
+
+  const accessTokenPayload = {
+    email,
+    sub: 'accessToken',
+  };
+
+  const refreshTokenPayload = {
+    email,
+    sub: 'refreshToken',
+  };
+
+  const accessToken = createToken(accessTokenPayload);
+  const refreshToken = createToken(refreshTokenPayload, false);
+  // await prisma.user_security.update({
+  //   where: { user_id: id },
+  //   data: { refresh_token: refreshToken },
+  // });
```

Comment:
Is this a template for the future?

Reply:
Yes. In last commit I added update refresh token logic when user is logged. Please cheack this code.